infra/aws/index.ts

```diff
@@ -12,173 +10,67 @@ const defaultTags = {
-);
+const coreInfraStack = new pulumi.StackReference(`passportxyz/core-infra/${stack}`);
 
-const stakingEnvironment = secretsManager
+const passportXyzAppEnvironment = secretsManager
   .getEnvironmentVars({
     vault: "DevOps",
-    repo: "passport", // TODO: this should be moved to id-staking-v2
+    repo: "passport",  // TODO: this should be moved to id-staking-v2
     env: stack,
-    section: "staking", // TODO: this section should be `app` ?
+    section: "staking",  // TODO: this section should be `app` ?
   })
   .reduce((acc, { name, value }) => {
     acc[name] = value;
     return acc;
   }, {} as Record<string, string | pulumi.Output<any>>);
 
-passportXYZDomain.apply((domainName) => {
-  const name = `${prefix}.${domainName}`;
-
-  const amplifyApp = new aws.amplify.App(
-    name,
-    {
-      name: name,
-      repository: APP_GITHUB_URL,
-      platform: "WEB_COMPUTE",
-      buildSpec: `version: 1
-applications:
-  - frontend:
-      phases:
-        preBuild:
-          commands:
-            - yarn install
-        build:
-          commands:
-            - yarn run build
-      artifacts:
-        baseDirectory: .next
-        files:
-          - '**/*'
-      cache:
-        paths:
-          - .next/cache/**/*
-          - node_modules/**/*
-    appRoot: app
-  `,
-      customRules: [
-        {
-          source: "/<*>",
-          status: "404",
-          target: "/index.html",
-        },
-      ],
-      environmentVariables: {
-        AMPLIFY_DIFF_DEPLOY: "false",
-        AMPLIFY_MONOREPO_APP_ROOT: "app",
-        ...stakingEnvironment,
-      },
-      tags: {
-        Name: name,
-        ...defaultTags,
-      },
-    },
-    { protect: true }
-  );
-
-  const branch = new aws.amplify.Branch(`${name}-${stakingBranches[stack]}`, {
-    appId: amplifyApp.id,
-    branchName: stakingBranches[stack],
-    displayName: stakingBranches[stack],
-    // stage: amplifyStage[stack],
-    ttl: "5",
-  });
-
-  const webHook = new aws.amplify.Webhook(`${name}-${stakingBranches[stack]}`, {
-    appId: amplifyApp.id,
-    branchName: stakingBranches[stack],
-    description: `trigger build from branch ${stakingBranches[stack]}`,
-  });
-
-  const domainAssociation = new aws.amplify.DomainAssociation(name, {
-    appId: amplifyApp.id,
+const amplifyAppInfo = coreInfraStack.getOutput("newPassportDomain").apply((domainName) => {
+  const prefix = "stake";
+  const amplifyAppConfig: amplify.AmplifyAppConfig = {
+    name: `${prefix}.${domainName}`,
+    githubUrl: PASSPORT_APP_GITHUB_URL,
+    githubAccessToken: PASSPORT_APP_GITHUB_ACCESS_TOKEN_FOR_AMPLIFY,
     domainName: domainName,
-    subDomains: [
+    cloudflareDomain: CLOUDFLARE_DOMAIN,
+    cloudflareZoneId: CLOUDFLARE_ZONE_ID,
+    prefix: prefix,
+    branchName: passportBranches[stack],
+    environmentVariables: passportXyzAppEnvironment,
+    tags: { ...defaultTags, Name: `${prefix}.${domainName}` },
+    enableBasicAuth: false,
```

Comment:
enableBasicAuth, basicAuthUsername, basicAuthPassword could be removed (those could be at least the default in the core-infra )